### ISSUE TITLE
EN support Android version 6.0 (API version 23) or higher

### ISSUE THREAD

**Issue by oyasan-ba2** [Covid-19Radar/Covid19Radar#319]
### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [x] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
Install Covid19Radar.Android onto less than Android 6.0 and finish to install successfully.

### Any log messages given by the failure
None

### Expected/desired behavior
Install fail because EN only support Android version 6.0 (API version 23) or higher

### OS and Version?
Less than Android 6.0 such as Android 5.0 or so.

**Comment by oyasan-ba2** [comment]
Sorry!. I have a target device trouble. Please wait around 1 hour to check and PR.

**Comment by oyasan-ba2** [comment]
Here is the reason Why:

For this app to work on a device, the device must be running Android version 6.0 (API 23) or higher (in Android-Exposure-Notification-API-documentation-v1.3.2.pdf page 3).
https://static.googleusercontent.com/media/www.google.com/en//covid19/exposurenotifications/pdfs/Android-Exposure-Notification-API-documentation-v1.3.2.pdf

Major Bluetooth differences between Android 5.0 (API 21) and Android 6.0 (API 23) are follows:

Secure Bluetooth communication that the device identify is difficult to know Bluetooth address.
When a device running Android 6.0 (API level 23) initiates a background Wi-Fi or Bluetooth scan, the operation is visible to external devices as originating from a randomized MAC address.
https://developer.android.com/about/versions/marshmallow/android-6.0-changes?authuser=1#behavior-hardware-id

**Comment by oyasan-ba2** [comment]
This issue is closed because this issue was fixed by Fix minSdkVersion 23 #356
